src/components/controls/km/km-validation-errors.vue

```diff
@@ -0,0 +1,136 @@
+<template>
+  <div class="scbd-controls km-validation-errors document-validation">
+    <div
+      v-if="reports && reports.isAnalyzing"
+      class="alert alert-info"
+      role="alert"
+    >
+      <i class="bi bi-gear"></i> {{ t("analyzingDocument") }}
+    </div>
+    <div
+      v-if="reports && reports.isSaving"
+      class="alert alert-info"
+      role="alert"
+    >
+      <i class="bi bi-gear"></i> {{ t("savingDocument") }}
+    </div>
+    <div
+      v-if="
+        reports && !reports.isAnalyzing && !reports.isSaving && !reports.errors
+      "
+      class="alert alert-success"
+      role="alert"
+    >
+      {{ t("valid") }}
+    </div>
+    {{ reports }} --- {{ internalReports }}
+    <div v-if="reports && !reports.isAnalyzing && !reports.isSaving">
+      <div
+        v-if="!hideErrors && reports && reports.errors && reports.errors.length"
+        class="alert alert-warning"
+        role="alert"
+      >
+        <button
+          type="button"
+          @click="onButtonClose"
+          aria-label="Close"
+          class="
+            position-absolute
+            top-0
+            end-0
+            m-2
+            bg-transparent
+            border-0
+            fs-5
+            text-secondary
+          "
+        >
+          <span aria-hidden="true">&times;</span>
+        </button>
+        <strong>
+          {{ t("contains") }} {{ reports.errors.length }} {{ t("errors") }}
+        </strong>
+
+        <ul>
+          <li
+            v-for="error in reports.errors"
+            :title="error.parameters"
+            :key="error"
+          >
+            <span>{{ t(getTranslation(error)) }}</span>
+            <ul v-if="error.properties">
+              <li
+                v-for="property in error.properties"
+                :title="error.parameters"
+                :key="property"
+              >
+                <a
+                  class="text-decoration-none"
+                  href="#"
+                  @click.prevent="jumpTo($event, property.property)"
+                >
+                  {{ getLabel(property.property) }}
+                </a>
+              </li>
+            </ul>
+            <span v-if="!error.properties">
+              <a
+                class="text-decoration-none"
+                href="#"
+                @click.prevent="jumpTo($event, error.property)"
+              >
+                {{ getLabel(error.property) }}
+              </a>
+            </span>
+          </li>
+        </ul>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script setup>
+import { useAttrs, defineEmits, defineProps, computed, onBeforeMount, inject } from "vue";
+
+const props = defineProps({
+  reports: { type: Object, required: true },
+});
+const $emits = defineEmits(["onJumpTo"]);
+const hideErrors = computed({
```

Comment:
allow-show-errors props default true
bool ref hideErrors for dismiss errors
showErrors computed(()=>!hideErrors && reports && reports.errors && reports.errors.length && props.allow-show-errors)

Reply:
replaced the component with 4 seperate props. Replaced the existing computed property and added hideError ref.